src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsEmailPreferences/SettingsEditSettingsEmailPreferences.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from "react"
+import * as React from "react"
+import { Banner, Box, Flex, Select, Text, useToasts } from "@artsy/palette"
+import { useSystemContext } from "v2/System/SystemContext"
+import { createFragmentContainer, graphql } from "react-relay"
+import { UpdateUserEmailPreferencesMutation } from "../../../../../../Components/UserSettings/UserEmailPreferences/UserEmailPreferencesMutation"
```

Comment:
Can we fix this relative path to be absolute?

Reply:
Updated.